styles/_theme.scss

```diff
@@ -12,6 +12,10 @@
   display: hidden;
 }
 
+.usa-graphic-list {
+  font-family: Source Sans Pro Web, Helvetica Neue, Helvetica, Roboto, Arial, sans-serif;
```

Comment:
on the one hand, font families with spaces should be wrapped in quotation marks: https://developer.mozilla.org/en-US/docs/Web/CSS/font-family#:~:text=It%20is%20good%20practice%20to%20quote%20font%20family%20names%20that%20contain%20white%20space%2C%20digits%2C%20or%20punctuation%20characters%20other%20than%20hyphens

but more importantly, when using USWDS, it's not appropriate to set font-family strings like this "hardcoded" -- we're urged to use the USWDS design tokens and utilities: https://designsystem.digital.gov/utilities/font-size-and-family/

`@include u-font-family('sans')`
Means that if we ever change sans serif font in our theme settings file, (or body, or heading, depending on which token you use in that function), it updates everywhere, rather than needing a find-and-replace across the whole css folder.

Please use USWDS tokens and utilities for as many overrides as possible.

Reply:
And now I know where that documentation is!  :)